documentation/concepts/notifications/notification-concept.adoc

```diff
@@ -127,30 +127,32 @@ A task provider operation::
 
     We don't distinguish between UI operations and sync adapter operations. We should
     check if a "notification relevant" field has been updated. This way we can avoid
-    notification updates in case only a field like "title" has been changed.
+    notification updates in case only a field like "description" has been changed.
```

Comment:
Oh, true. Notifications show the title :-o

Reply:
This is one "bug" I am suffereing right now but that is of minor importance as it happens only rarely.